Makefile

```
@@ -122,33 +126,48 @@ precommit: build unit-test lint
 deepcopy-gen:
 	deepcopy-gen -i ./pkg/apis/aadpodidentity/v1/ -o . -O aadpodidentity_deepcopy_generated -p aadpodidentity
 
+.PHONY: docker-buildx-builder
+docker-buildx-builder:
+	docker run --rm --privileged docker/binfmt:820fdd95a9972a5308930a2bdfb8573dd4447ad3
+	if ! DOCKER_CLI_EXPERIMENTAL=enabled docker buildx ls | grep -q container-builder; then\
+		DOCKER_CLI_EXPERIMENTAL=enabled docker buildx create --name container-builder --use;\
+	fi
```

`DOCKER_CLI_EXPERIMENTAL` is already exported above

```suggestion
	if ! docker buildx ls | grep -q container-builder; then \
		docker buildx create --name container-builder --use; \
	fi
```